### DISCUSSION TITLE
How to not render the content scripts when using DOM for Button.

### DISCUSSION THREAD

**Discussion by Rishav1707** [open-sauced-craftwork/browser-extension#40] [Q&A]
https://user-images.githubusercontent.com/97666287/227239915-bcb10bf8-f115-4360-a58b-5aeb754694e4.mp4

## It first renders the content script also i.e "Invalid User"

**Comment by anush008** [comment]
You'll have to provide additional information about your implementation in the content-script to get help with the issue.

**Reply by Rishav1707** [reply]
![Manifest](https://user-images.githubusercontent.com/97666287/227276358-ad5fd418-50d8-4fad-83cc-8573f5ba6a4c.png)

**Comment by Rishav1707** [comment]
## Content.jsx file

![content](https://user-images.githubusercontent.com/97666287/227276672-ed0c611b-59e2-4416-abb8-8d951cfb9800.png)

**Reply by anush008** [reply]
This won't help, since it's just the boilerplate code. You'll have to provide specific details about your button rendering implementation.

**Reply by Rishav1707** [reply]
Shall I DM you?

**Reply by anush008** [reply]
Okay. It's Anush#5038 on Discord.

**Reply by Rishav1707** [reply]
I have sent you a request
